docs/linkify-angular-ts.md

```diff
@@ -0,0 +1,251 @@
+
+
+## Demo
+
+View all the directives in action at [https://anthonynahas.github.io/ngx-linkifyjs](https://anthonynahas.github.io/ngx-linkifyjs)
+
+## Installation
+
+
+Install above dependencies via *npm*. 
+
+Now install `ngx-linkifyjs` via:
+```shell
+npm i -s ngx-linkifyjs
+```
+
+---
+##### SystemJS
+>**Note**:If you are using `SystemJS`, you should adjust your configuration to point to the UMD bundle.
+In your systemjs config file, `map` needs to tell the System loader where to look for `ngx-linkifyjs`:
+```js
+map: {
+  'ngx-linkifyjs': 'node_modules/ngx-linkifyjs/bundles/ngx-linkifyjs.umd.js',
+}
+```
+---
+
+Once installed you need to import the main module:
+```js
+import { NgxLinkifyjsModule } from 'ngx-linkifyjs';
+```
+The only remaining part is to list the imported module in your application module. The exact method will be slightly
+different for the root (top-level) module for which you should end up with the code similar to (notice ` NgxLinkifyjsModule .forRoot()`):
+```typescript
+import { NgxLinkifyjsModule } from 'ngx-linkifyjs';
+
+@NgModule({
+  declarations: [AppComponent, ...],
+  imports: [NgxLinkifyjsModule.forRoot(), ...],  
+  bootstrap: [AppComponent]
+})
+export class AppModule {
+}
+```
+
+Other modules in your application can simply import ` NgxLinkifyjsModule `:
+
+```js
+import { NgxLinkifyjsModule } from 'ngx-linkifyjs';
+
+@NgModule({
+  declarations: [OtherComponent, ...],
+  imports: [NgxLinkifyjsModule, ...], 
+})
+export class OtherModule {
+}
+```
+
+
+## Usage
+
+
+Once the library is imported, you can use its components, directives and pipes in your Angular application:
+
+### Pipe
+
+`{{text | linkify}}`
+
+```html
+<span [innerHTML]="'Linkify the following URL: https://github.com/anthonynahas/ngx-linkifyjs and share it <3' | linkify"></span>
+```
+
+**result**: Linkify the following URL: [https://github.com/anthonynahas/ngx-linkifyjs](https://github.com/anthonynahas/ngx-linkifyjs) and share it <3
+
+### Service
+
+Inject the `NgxLinkifyjsService` service
+
+```typescript
+import {NgxLinkifyjsService, Link, LinkType} from 'ngx-linkifyjs';
+
+constructor(public linkifyService: NgxLinkifyjsService) {
+ } 
+}
+```
+
+#### Methods
+
+<a name="linkify_method"/>
+
+#### linkify _(text: string): string_ 
+
+Convert a basic text string to a valid linkified text
+
+**Params**
+
+*  **`text`** : _`String`_ Text to linkify --> to convert with links
+
+**Returns** _`String`_  converted text with links
+
+
+```typescript
+import {NgxLinkifyjsService, Link, LinkType} from 'ngx-linkifyjs';
+
+constructor(public linkifyService: NgxLinkifyjsService) {
+  this.linkifyService.linkify('For help with GitHub.com, please email support@github.com');
+  // result --> see below
+ } 
+}
+```
+
+```typescript
+'For help with <a href=\"http://github.com\" class=\"linkified\" target=\"_blank\">GitHub.com</a>, please email <a href=\"mailto:support@github.com\" class=\"linkified\">support@github.com</a>'
```

Comment:
I don't think the double quotes should be escaped since this is a single-quoted string
```suggestion
'For help with <a href="http://github.com" class="linkified" target="_blank">GitHub.com</a>, please email <a href="mailto:support@github.com" class="linkified">support@github.com</a>'
```